website/content/docs/services/discovery/dns-forwarding/enable.mdx

```diff
@@ -0,0 +1,442 @@
+
+<CodeBlockConfig filename="/etc/dnsmasq.conf">
+
+```plaintext
+# Accept DNS queries only from hosts whose address is on a local subnet.
+#local-service
+
+# Don't poll /etc/resolv.conf for changes.
+#no-poll
+
+# Don't read /etc/resolv.conf. Get upstream servers only from the command
+# line or the dnsmasq configuration file (see the "server" directive below).
+#no-resolv
+
+# Specify IP address(es) of other DNS servers for queries not handled
+# directly by consul. There is normally one 'server' entry set for every
+# 'nameserver' parameter found in '/etc/resolv.conf'. See dnsmasq(8)'s
+# 'server' configuration option for details.
+#server=1.2.3.4
+#server=208.67.222.222
+#server=8.8.8.8
+
+# Set the size of dnsmasq's cache. The default is 150 names. Setting the
+# cache size to zero disables caching.
+#cache-size=65536
+```
+
+</CodeBlockConfig>
+
+## BIND
+
+[BIND](https://www.isc.org/bind/) is a suite of software for interacting with the Domain Name System (DNS). Its most prominent component, `named`, short for name daemon, performs both of the main DNS server roles, acting as an authoritative name server for DNS zones and as a recursive resolver in the network.
+
+<Note>
+
+If your distribution uses systemd, you may need to disable `systemd-resolved` prior to following this example. 
+ 
+</Note>
+
+You can configure the BIND service to send `.consul` domain queries to Consul by:
+- creating a configuration file for `named` with `DNSSEC` disabled
+- creating a zone configuration file to manage the `.consul` domain 
+
+### Named configuration file
+
+Edit the `/etc/named.conf` to configure your BIND instance. Remember to disable `DNSSEC` so that Consul and BIND can communicate. Also add an `include` section to include the zone file you will create in the next step.
+
+The following example shows a BIND configuration with `DNSSEC` disabled.
+
+<CodeBlockConfig filename="/etc/named.conf">
+
+```plaintext
+options {
+  listen-on port 53 { 127.0.0.1; };
+  listen-on-v6 port 53 { ::1; };
+  directory       "/var/named";
+  dump-file       "/var/named/data/cache_dump.db";
+  statistics-file "/var/named/data/named_stats.txt";
+  memstatistics-file "/var/named/data/named_mem_stats.txt";
+  allow-query     { localhost; };
+  recursion yes;
+
+  dnssec-enable no;
+  dnssec-validation no;
+
+  /* Path to ISC DLV key */
+  bindkeys-file "/etc/named.iscdlv.key";
+
+  managed-keys-directory "/var/named/dynamic";
+};
+
+include "/etc/named/consul.conf";
+```
+
+</CodeBlockConfig>
+
+### Zone file
+
+Set up a zone for your Consul managed records in `consul.conf`.
+
+<CodeBlockConfig filename="/etc/named/consul.conf">
+
+```dns-zone-file
+zone "consul" IN {
+  type forward;
+  forward only;
+  forwarders { 127.0.0.1 port 8600; };
+};
+```
+
+</CodeBlockConfig>
+
+## Unbound
+
+[Unbound](https://www.unbound.net/) is a validating, recursive, caching DNS resolver. It is designed to be fast and lean and incorporates modern features based on open standards.
```

Comment:
This text is verbatim from the Unbound website. Suggest updating to something like

```suggestion
Use [Unbound](https://www.unbound.net/) when you need a fast and lean DNS resolver for Linux and macOS.
```